lib/traject/config/folio_config.rb

```diff
@@ -146,18 +146,18 @@ def holdings(record, context)
   end
 end
 
-# TODO: is this the right place to get this from? use instanceRecord or holdingsRecord instead?
-# when are these timestamps updated in FOLIO?
 to_field 'date_cataloged' do |record, accumulator|
-  timestamp = record.record.dig('metadata', 'createdDate')
+  timestamp = record.instance['catalogedDate'] || record.instance.dig('metadata', 'createdDate')
```

Comment:
It looks like when there is no cataloged data in FOLIO, the field is not returned in the response for something like `GET https://${OKAPI_URL}/inventory/instances?query=hrid=="in00000000224"`. I don't think we should use createdDate if `record.instance['catalogedDate']` is missing because it could put way too many things at the top of the sort when we batch-load records into the system, making "new to the library" sort of useless.